benchmarks/src/main/scala/zio/StreamBenchmarks.scala

```diff
@@ -277,6 +277,25 @@ class StreamBenchmarks {
     unsafeRun(result)
   }
 
+  @Benchmark
+  def zioRepeatZIOChunkOption: Long = {
+    var index = 0
+
+    val result =
+      ZStream.repeatZIOChunkOption {
```

Comment:
I think we discussed this before, but I think we need to make this suspend automatically to avoid the `ZIO.suspendSucceed` below